utils/ioc_utils.py

```diff
@@ -8,41 +9,58 @@
 import logging
 
 
+def _add_ioc_directory(device_info):
+    logging.info("Creating support directory {0}".format(device_info.ioc_path))
+    mkdir(device_info.ioc_path())
+    return device_info.ioc_path()
+
+
 def _run_ioc_template_setup(device_info, device_count):
     """
     Runs the EPICS perl scripts associated with IOC creation. Passes in the IBEX type flag to use our own templates
     found in C:\\Instrument\\Apps\\EPICS\\base\\master\\templates
 
     Args:
-        device_info: Name-based information about the device
-        device_count: How many IOC apps to generate
+        device_info (DeviceInfoGenerator): Name-based information about the device
+        device_count (int): How many IOC apps to generate
+    Returns:
+        ioc_path (Str): Path to the IOC directory
     """
-    if device_count > 99:
-        raise ValueError("Cannot generate more than 99 IOCs for a single device")
+    if device_count > 9:
+        raise ValueError("Cannot generate more than 9 IOCs for a single device")
```

Comment:
There are some devices where we will require more than 9 IOCs (e.g. danfysik, galil). The previous limit (99) was more appropriate. We should put the limit back to 99.

Reply:
For creating a component for the ioc, the action from `ioc_utils` is the `create_ioc` method. This is used to call `_run_ioc_template_setup`. In the `create_ioc` method, it performs a check to ensure the device count is within the range 1 to 9. This means you could never generate 99 template iocs directories.

Should I modify the `create_ioc` method to support 99? How were you ever able to generate 99 iocs?

Reply:
The device generator certainly used to work with 2-digit IOC numbers. It may have become broken during one of the refactors. We should make it work for 2-digit ioc numbers unless it's very difficult for some reason, and add a test to make sure it doesn't break in this way in future.